scipy/integrate/_bvp.py

```diff
@@ -491,7 +487,7 @@ def solve_newton(n, m, h, col_fun, bc, jac, y, p, B, bvp_tol):
             break
 
         if (np.all(np.abs(col_res) < tol_r * (1 + np.abs(f_middle))) and
-                np.all(bc_res < tol_bc)):
```

Comment:
Did we have a mistaker here of not using `abs`?

Reply:
Yes, I changed `bc_res` to `np.abs(bc_res)`.